homework/list.js

```diff
@@ -1,79 +1,83 @@
 'use strict';
+function ToDoList(){
+    this._listNoteContent = [];
+    this._wrapper = document.querySelector('#note-list-wrapper');
+}
+ToDoList.prototype = {
+    start: function() {
+	this.fetchList(function(data) {
+	    this.updateList(data);
+	    this.drawList();
+	    this.preloadFirstNote();
+	}.bind(this));
+	window.addEventListener('click', function(event) {
+	    this.onNoteOpen(event);
```

Comment:
It's a convention to prefix method with `on` for the corresponding function, so it should be renamed as 'onClick'. And, to set `this` as the listener is better than an anonymous function.